pkg/agent/agent.go

```diff
@@ -168,7 +168,7 @@ func (i *Initializer) GetWireGuardClient() wireguard.Interface {
 // setupOVSBridge sets up the OVS bridge and create host gateway interface and tunnel port
 func (i *Initializer) setupOVSBridge() error {
 	if err := i.ovsBridgeClient.Create(); err != nil {
-		klog.Error("Failed to create OVS bridge: ", err)
+		klog.Errorf("Failed to create OVS bridge")
```

Comment:
Avoiding duplicated logs. Since the err will be logger by the Initialize() function as well, so just kept the customer error msg here, while the actual error will be returned back to the caller